locales/en/data-classes.ftl

```diff
@@ -78,6 +80,7 @@ parenting-plans = Parenting plans
 partial-credit-card-data = Partial credit card data
 passport-numbers = Passport numbers
 password-hints = Password hints
+password-strengths = Password strengths
```

Comment:
This looks like a weird category of data class 🤷🏼

Reply:
Agreed! I have no idea why a company would store that in a database.